varnish/default.vcl

```diff
@@ -54,8 +58,13 @@ sub vcl_recv {
         return (pass);
     }
 
-    # Bypass shopping cart, checkout and search requests
-    if (req.url ~ "/checkout" || req.url ~ "/catalogsearch") {
+    # Bypass customer, shopping cart, checkout
+    if (req.url ~ "/customer" || req.url ~ "/checkout") {
+        return (pass);
+    }
```

Comment:
Bijzonder eigenlijk dat dit hier in zit, terwijl je ook gewoon in Magento de juiste Cache-Control headers kunt setten.. 🤷